readme.md

```diff
@@ -1284,6 +1284,93 @@ echo $faker->bankAccountNumber // "LT300848876740317118"
 echo $faker->personalIdentityNumber; // "140190-12301"
 ```
 
+### `Faker\Provider\ms_MY\Address`
+
+```php
+<?php
+
+// Generates a random Malaysian street address
+echo $faker->streetAddress; // "No. 123, Jalan Utama 7"
+
+// Generates a random Malaysian township
+echo $faker->township; // "Taman Bahagia"
+
+// Generates a random Malaysian town address with matching postcode and state
+echo $faker->townState; // "55100 Bukit Bintang, Kuala Lumpur"
+
+// Generates a random Malaysian address
+echo $faker->address; // "No. 123, Jalan Utama 7, Taman Bahagia, 55100 Bukit Bintang, Kuala Lumpur"
```

Comment:
please don't document formatters that are not specific to your locale

Reply:
ok they have been removed as requested.